src/core_atmosphere/physics/physics_mmm/bl_ysu.F

```diff
@@ -893,6 +892,7 @@ subroutine bl_ysu_run(ux,vx,tx,qvx,qcx,qix,nmix,qmix,p2d,p2di,pi2d,     &
        endif
      enddo
    enddo
+
```

Comment:
between commits "82dc4d0e4d8fd182dc4d0e4d8fd1" and "5e3412e8b8972ba", I did not see any updates related to bl_ysu.F, except for the initialization of we.

Can you clarify the changes to bl_ysu.F relative to what? Thanks.

Reply:
Just removing the initialization of `we` and removing the added blank line 895 should be all that's needed.

Reply:
I compared the develop branch at hash 82dc4d0e4d8f which is the reference hash I used to add the Noahmp LSM. At that specific hash, bl_ysu.F from the develop and NSF-MPAS/lsm_noahmp branch are the same.

I also compared the develop branch at hash 82dc4d0e4d8f against the branch NSF-MPAS/lsm_noahmp at hash 5e3412e8b897 which corresponds to the last commit I made yesterday evening. Here, I get the following differences:

diff ./../../../../../../../MPAS-Dev/MPAS-Model/src/core_atmosphere/physics/physics_mmm/bl_ysu.F .
564d563
<      we(i)     = 0.
895a895
> 

So at that hash, we already have initialized "we" in the develop branch and I do have a blank line in NSF-MPAS/lsm_noahmp. Note that I had previously removed the initialization of "we" in NSF-MPAS/lsm_noahmp (see hash 5a1154f78fd4. 

Do you agree that what, in order for bl_ysu.F to be the same in the develop branch and NSF-MPAS/lsm_noahmp at hash 82dc4d0e4d8f, I simply need to:

1) re-add the initialization of we, and 2) delete the extra blank line?

Thanks.

Reply:
The commits that you are referring to add and then remove the initialization of we. I am not familiar with interactive rebase. As you suggested, I would really appreciate if you could handle the rebase to have the order of successive commits as it is supposed to be. It may be faster as well? Thanks.

Reply:
No problem -- I've just rebased and force-pushed the `NSF-MPAS/lsm_noahmp` branch.

Reply:
Many thanks.